settings.toml

```diff
@@ -0,0 +1,8 @@
+[default.service]
```

Comment:
Why "default" in the block name? Does it have any special meaning?

Reply:
It is needed when using the `FlaskDynaConf`, extension of DynaConf specific for Flask

https://www.dynaconf.com/flask/#settings-files

It needs to have an environment associated to that configuration